packages/sourcecred/src/plugins/discord/mirror.js

```diff
@@ -36,7 +36,10 @@ export class Mirror {
       try {
         await this.addMessages(channel.id);
       } catch (e) {
-        console.warn(e);
+        const warn = e.message.includes("403")
```

Comment:
I might not be particularly satisified with this check and also skeptical about the message being part of the error, do we have any other object that we can be sure of, like status, or any check as to not break the execution if the message was an optional at times, I imagine a scenario where either the api changes for these messages, or a time where the internet cuts off and the execution is halted because of "Cannot read property includes of null"

Reply:
we throw this error ourselves, so it should be ok
https://github.com/sourcecred/sourcecred/blob/main/packages/sourcecred/src/plugins/discord/fetcher.js#L229

I will add null safety though, since who knows what could find it's way up the promise chain.

Reply:
Awesome!